Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -169,6 +182,12 @@ class BrowserViewController: UIViewController {
         dismissVisibleMenus()
 
         coordinator.animate(alongsideTransition: { context in
+            let portrateImage = UIImage(named: "searchBackgroundImage")
```

Comment:
How about we move  this block into `searchBackgroundImageView` , looks like it repeats itself

Reply:
Oh, It cannot be moved as it has to be checked everytime. 

How about putting it in a function?